packages/tools/devtools/devtools-view/src/components/OpLatencyView.tsx

```diff
@@ -44,6 +51,109 @@ const useStyles = makeStyles({
 export function OpLatencyView(): React.ReactElement {
 	const styles = useStyles();
 
+	const messageRelay = useMessageRelay();
+	const [durationOutboundBatchingData, setDurationOutboundBatchingData] =
+		React.useState<GraphDataSet>({
+			graphType: "line",
+			schema: {
+				displayName: "Duration Outbound",
+				uuid: "durationOutbound",
+				xAxisDataKey: "timestamp",
+				yAxisDataKey: "duration",
+			},
+			data: [],
+		});
+	const [durationNetworkData, setDurationNetworkData] = React.useState<GraphDataSet>({
+		graphType: "line",
+		schema: {
+			displayName: "Duration Network",
+			uuid: "durationNetwork",
+			xAxisDataKey: "timestamp",
+			yAxisDataKey: "duration",
+		},
+		data: [],
+	});
+	const [durationInboundToProcessingData, setDurationInboundToProcessingData] =
+		React.useState<GraphDataSet>({
+			graphType: "line",
+			schema: {
+				displayName: "Duration Inbound",
+				uuid: "durationInbound",
+				xAxisDataKey: "timestamp",
+				yAxisDataKey: "duration",
+			},
+			data: [],
+		});
+	const unsampledTelemetry = localStorage.getItem("Fluid.Telemetry.DisableSampling");
+	// Render the text conditionally
+	const renderText = unsampledTelemetry !== "true";
+
+	React.useEffect(() => {
+		/**
+		 * Handlers for inbound messages.
+		 */
+		const inboundMessageHandlers: InboundHandlers = {
+			[TelemetryEvent.MessageType]: async (untypedMessage) => {
+				const message = untypedMessage as TelemetryEvent.Message;
+				const eventContents = message.data.event.logContent;
+				// We ignore the ops related to round trip time or summarizer as Op latency is focused on ops generated by the user
+				if (!eventContents.eventName.endsWith("OpRoundtripTime")) {
+					return false;
+				}
+
+				if (eventContents.clientType === "noninteractive/summarizer") {
+					return false;
+				}
+
+				setDurationOutboundBatchingData((currentData) => {
+					const newDataPoint = {
+						outboundtimestamp: message.data.event.timestamp,
+						durationOutboundBatching: Number(eventContents.durationOutboundBatching),
```

Comment:
I think all of these updates need to now create points with `timestamp` and `duration` as properties instead of the old names, right?